testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/MiniAppDisplayViewModel.kt

```diff
@@ -42,7 +42,7 @@ class MiniAppDisplayViewModel constructor(
     ) = viewModelScope.launch(Dispatchers.IO) {
         try {
             _isLoading.postValue(true)
-            miniAppDisplay = miniapp.create(appId, miniAppMessageBridge, miniAppNavigator)
```

Comment:
Then, should we remove the sample app implementation of `miniAppNavigator` including the secondary webview? Because it's to upto HostApp if they wants to implement secondary webview or other technology with providing non-null `miniAppNavigator`.

Reply:
The miniAppNavigator in demo app and the default one are different. We should keep the miniAppNavigator in our Demo app to show user how to send data back.